apps/sports-helsinki/src/pages/venues/[venueId]/index.tsx

```diff
@@ -7,26 +11,25 @@ import type {
 import type { GetStaticPropsContext, NextPage } from 'next';
 import React, { useContext } from 'react';
 import { Page as RHHCPage } from 'react-helsinki-headless-cms';
-import Navigation from '../../../common-events/components/navigation/Navigation';
-import AppConfig from '../../../domain/app/AppConfig';
-import getSportsStaticProps from '../../../domain/app/getSportsStaticProps';
-import FooterSection from '../../../domain/footer/Footer';
-import serverSideTranslationsWithCommon from '../../../domain/i18n/serverSideTranslationsWithCommon';
-import MatomoWrapper from '../../../domain/matomoWrapper/MatomoWrapper';
-import VenuePageContainer from '../../../domain/venue/VenuePageContainer';
-import { getLocaleOrError } from '../../../utils/routerUtils';
+import AppConfig from 'domain/app/AppConfig';
```

Comment:
how thse non-relative paths can work?

Reply:
I have no idea 😄 It seems like the domain-path as an absolute path always works. Some constants and types does not, because it has multiple alternatives. Do we want to use these, or should I go through all those thousands of them? 😄 These were added by the Vscode and I just went from there where the fence was the lowest

Reply:
https://beqode.com/blog/absolute-vs-relative-import-paths-nodejs

Reply:
absolute path == relative path from the root